rskj-core/src/main/resources/config/main.conf

```diff
@@ -55,6 +55,19 @@ peer {
     networkId = 775
 }
 
+rest {
```

Comment:
do we need these network specific configs? it seems to me that they are duplicates on the one defined in reference.conf